.github/workflows/go.yml

```diff
@@ -35,6 +36,7 @@ jobs:
       matrix:
         go-version: [1.19.x]
         os: [ubuntu-22.04]
+        transport: [ "", "tcp", "quic" ]
```

Comment:
it tests it separately for each protocol?
what happens with the "" protocol?

Reply:
If -onlyTransports is not specified, then mixes listen on all protocols.

When outgoing connections are established (server/internal/outgoing/outgoing_conn.go:164) the Address order depends on the order populated from ranging over the MixDescriptor Addresses (core/pki/descriptor.go:74)